plugins/bucketize_procedure.cc

```diff
@@ -177,14 +197,14 @@ run(const ProcedureRunConfig & run,
         
         ExcAssert(higherBound <= rowCount);
 
-        cerr << "Bucket " << bucketName << " from " << lowerBound
-             << " to " << higherBound << endl;
+        //cerr << "Bucket " << mappedRange.first << " from " << lowerBound
+        //     << " to " << higherBound << endl;
 
         ML::run_in_parallel_blocked(lowerBound, higherBound, applyFct);
     }
 
     // record remainder
-    accum.forEach([&] (vector<pair<RowName, vector<cell>>> * rows)
+    accum.forEach([&] (vector<pair<RowName, vector<Cell> > > * rows)
```

Comment:
Why the extra whitespaces? Is it auto formatting? I think old c++ compilers required it, is that the reason?